sample-app/src/main/java/com/squareup/contour/sample/widget/PaddingAdjusterWidget.kt

```diff
@@ -0,0 +1,111 @@
+package com.squareup.contour.sample.widget
+
+import android.graphics.Rect
+import android.widget.SeekBar
+import androidx.appcompat.widget.AppCompatSeekBar
+import androidx.appcompat.widget.AppCompatTextView
+import com.squareup.contour.ContourLayout
+import com.squareup.contour.sample.SampleActivity
+import kotlin.contracts.ExperimentalContracts
+
+@ExperimentalContracts
```

Comment:
I don't see any contract being used in this class?

Reply:
No, but it seems to be viral, and we get a lint error because we're using `SampleActivity` which is annotated...

Reply:
I see. Should we use `OptIn()` in `SampleActivity` in that case?

Reply:
I'm going to address the experimental annotations in a separate PR